src/contract/deploy.rs

```diff
@@ -102,6 +103,7 @@ where
 
 /// Builder for specifying options for deploying a linked contract.
 #[derive(Debug, Clone)]
+#[must_use = "deploy builers do nothing unless you `.deploy()` them"]
```

Comment:
Nice addition but again probably sh/could have been a separate PR.

Reply:
Yes, the issue was that I wanted to add it to the `MethodBuilder` and `ViewMethodBuilder` types as well and creating it as a separate PR would have been annoying.

Reply:
I actually added it to the MethodBuilder types and then noticed that we didn't have it on our other builder types so added it as well.